composer/content.md

```diff
@@ -75,6 +77,16 @@ Sometimes dependencies or Composer [scripts](https://getcomposer.org/doc/article
 
 -	Create your own image (possibly by extending `FROM composer`).
 
+**Note:** Docker introduced [multi-stage](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) builds in 17.05:
+
+-	Create your own image, and copy Composer from the official image into it:
+
+	```
```

Comment:
Add `dockerfile` as the "format" for this code block and Travis should be happy :+1:

Reply:
Done 👍